RepoDb/RepoDb/QueryBuilder.cs

```diff
@@ -41,7 +41,14 @@ public override string ToString()
         /// <returns>The current instance.</returns>
         public string GetString()
         {
-            return m_stringBuilder.ToString(1, m_stringBuilder.Length - 1);
+            if (m_stringBuilder.Length == 0)
```

Comment:
We are too micro managing the object here - but thanks for this :) - this could avoid the potential IndexOutOfRangeException.